### ISSUE TITLE
Multiple Configuration Transforms Not Working

### ISSUE THREAD

**Issue by MSWCRB** [golavr/ConfigurationTransform#2]
There is a bug with the current implementation that was identified by Dan Bullock on 9/19/2017 wherein the extensions is created a new AfterBuild section in the project file for each config transform in a project, which causes only the first transform to be applied in a scenario where a project has more than one config file that needs to be transformed. Dan also presents the solution there. Please fix it if you have time.

```
"The sample project is not correct. The sample defines multiple <AfterBuild> but when it actually can only be defined once. Each definition replaces the previous.

In .csproj it should be a single AfterBuild with each TransformXml having a condition:

<Target Name="AfterBuild">
<!--Generate transformed config in the output directory-->
<TransformXml Condition="Exists('data.$(Configuration).config')" Source="data.config" Destination="$(OutputPath)data.config" Transform="data.$(Configuration).config" />
<TransformXml Condition="Exists('plainxml.$(Configuration).config')" Source="plainxml.config" Destination="$(OutputPath)plainxml.config" Transform="plainxml.$(Configuration).config" />
</Target>
By Dan Bullock 9/19/2017"
```

**Comment by golavr** [comment]
@MSWCRB I checked it and you are right
thanks for the input

Steps to reproduce:
1. Open ConfigTest [sample](https://github.com/golavr/ConfigurationTransformSample) solution
2. Navigate to ConfigTest project
3. Transform App.config
4. Transform data.config
5. Transform plainxml.config
6. Open ConfigTest.csproj file

File contains 2 AfterBuild tasks, only the last one will be executed

`<Target Name="AfterBuild" Condition="Exists('data.$(Configuration).config')">
    
    <TransformXml Source="data.config" Destination="$(OutputPath)data.config" Transform="data.$(Configuration).config" />
  </Target>`

 `<Target Name="AfterBuild" Condition="Exists('plainxml.$(Configuration).config')">
    
    <TransformXml Source="plainxml.config" Destination="$(OutputPath)plainxml.config" Transform="plainxml.$(Configuration).config" />
  </Target>`

**Comment by MSWCRB** [comment]
The solution is to generate this (then both get executed):
`<Target Name="AfterBuild">  <TransformXml Condition="Exists('data.$(Configuration).config')" Source="data.config" Destination="$(OutputPath)data.config" Transform="data.$(Configuration).config" />
<TransformXml Condition="Exists('plainxml.$(Configuration).config')" Source="plainxml.config" Destination="$(OutputPath)plainxml.config" Transform="plainxml.$(Configuration).config" /> </Target>`

**Comment by golavr** [comment]
Fixed in v3.2
